### PR TITLE
initial commit of Solr Logs Integration

### PR DESCRIPTION
**Pull request by radu-gheorghe**
I've pushed a tiny change to RabbitMQ logs as well.

### CODE REVIEW

File: docs/integration/solr-logs.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  class="content-modal-image"
+  alt="Solr Queries Report"
+  src="../../images/agents/solr_logs_hits.png"
+  title="Solr Queries Report"
+/>
+
+Other built-in reports include:
+
+- **Errors**: breakdown of what's wrong: which nodes/classes/collections/etc generate errors
+- **Zookeeper**: logs produced by Solr's [Zookeeper](http://zookeeper.apache.org) client and other Zookeeper-related classes and threads. Look here for insights on SolrCloud's stability. For the Zookeeper ensemble itself, check out our [Zookeeper monitoring integration](./zookeeper.md)
```
Comment (otisg) [inline docs/integration/solr-logs.md:69]: http://zookeeper.apache.org => https://zookeeper.apache.org

File: docs/integration/solr-logs.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+- **Overseer**: logs produced by SolrCloud's [Overseer](https://lucene.apache.org/solr/8_6_0/solr-core/org/apache/solr/cloud/Overseer.html) thread and other Overseer activities (e.g. shard leader election)
```
Comment (otisg) [inline docs/integration/solr-logs.md:70]: Solr docs don't have a versionless URL that points to the latest version of docs?

**Review by otisg — APPROVED** [review]
LGTM, just minor stuff.

**Comment by otisg** [comment]
@radu-gheorghe once this is merged, could you please look at https://sematext.com/blog/solr-monitoring-made-easy-with-sematext/ and https://sematext.com/blog/solr-open-source-monitoring-tools/ and see if we can add links to the new docs from there, or maybe even include a screenshot of some of the new reports?

File: docs/integration/solr-logs.md (hunk 2 above)
Comment (radu-gheorghe) [reply docs/integration/solr-logs.md:70]: no, unfortunately. At least not that I know of :(

File: docs/integration/solr-logs.md (hunk 1 above)
Comment (radu-gheorghe) [reply docs/integration/solr-logs.md]: Fixed. I actually did notice it was http (via google), but it didn't occur to me that there is an https. On Monday morning, it's obvious, but not on Friday evening :p

**Comment by radu-gheorghe** [comment]
> @radu-gheorghe once this is merged, could you please look at https://sematext.com/blog/solr-monitoring-made-easy-with-sematext/ and https://sematext.com/blog/solr-open-source-monitoring-tools/ and see if we can add links to the new docs from there, or maybe even include a screenshot of some of the new reports?

Aha, good idea, thanks! The first link does mention logs, but I think it's worth doing a bit more there than just updating link and screenshot. There are some instructions there for Logagent that I think are worth updating. Let me create a task for it.

Second link is only about monitoring, almost no logs in sight. But I can add a reference, I think it would be useful anyway.

**Comment by otisg** [comment]
@adnanrahic please have a look so we can merge

**Review by adnanrahic — APPROVED** [review]
LGTM!

**Comment by otisg** [comment]
We can delete the branch, no?

**Comment by adnanrahic** [comment]
Yep.

**Pull request merged**